projects/components/src/quick-search/quick-search.component.ts

```diff
@@ -155,6 +167,12 @@ export class QuickSearchComponent implements OnInit {
     selectedItem: QuickSearchResultItem;
 
     ngOnInit(): void {
+        this.resultsGridId = this.idGenerator.generate();
+
+        this.ariaActivedescendantService.activeDescendantObservable.subscribe((id: string) => {
+            this.activeDescendantId = id;
+            this.changeDetectorRef.detectChanges();
```

Comment:
Why do we need to detectChanges? Aren't changes detected automatically on subscriptions?